### ISSUE TITLE
Aniyomi for pc/laptop version 

### ISSUE THREAD

**Issue by MHWizard** [jmir1/tachiyomi-mi#1840]
### Describe your suggested feature

please! Give some new aniyomi pc version. it well help so much for those whos mobail storage always full. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is a request regarding an extension, I should be opening an issue in the extension's repository.
- [X] I have updated the app to version **[0.16.4.3](https://github.com/aniyomiorg/aniyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by AureliusGemini** [comment]
nah, i use https://github.com/MustardChef/WSABuilds as WSA, WSL but Android.
https://github.com/alesimula/wsa_pacman is for download apk straight from websites and install

**Comment by AdityaJha012** [comment]
You can use mangayomi for it.. it has similar features like aniyomi... same interface and everything but for PC.. it is also available on github
